plugins/default/check_pod_disk_usage.sh

```diff
@@ -0,0 +1,106 @@
+################################
+# GLOBALS
+################################
+# Nagios exit codes
+NAGIOS_OK=0
+NAGIOS_WARN=1
+NAGIOS_CRIT=2
+NAGIOS_UNKNOWN=3
+ 
+################################
+# FUNCTIONS
+################################
+usage() {
+ echo "Usage: $0 -w <warn> -c <crit> [-m <usage|inodes>] [-h]"
+ echo " -w <warn> Warning threshold percentage (e.g. 80)"
+ echo " -c <crit> Critical threshold percentage (e.g. 90)"
+ echo " -m <mode> Check mode (usage or inodes)"
+ echo " -h        Show this message"
+ echo "Example: $0 -w 85 -c 95"
+}
+
+################################
+# ARGUMENT PROCESSING
+################################
+THRESH_WARN=
+THRESH_CRIT=
+CHECK_MODE=usage
+while getopts “hw:c:m:” OPTION
+do
+ case $OPTION in
+  h) usage; exit $NAGIOS_UNKNOWN ;;
+  w) THRESH_WARN=$OPTARG ;;
+  c) THRESH_CRIT=$OPTARG ;;
+  m) CHECK_MODE=$OPTARG ;;
+  ?) usage; exit $NAGIOS_UNKNOWN ;;
+ esac
+done
+ 
+if [[ -z $THRESH_WARN ]] || [[ -z $THRESH_CRIT ]]; then
+ usage; exit $NAGIOS_UNKNOWN
+fi
+
+TEMP_FILE=`mktemp`
+
+if [ "$CHECK_MODE" = "usage" ]; then
+	DF_CMD="df -x tmpfs"
```

Comment:
Based on the filesystems I've seen in our existing pods, all of the tmpfs ones wouldn't be getting data written to them normally, but I guess its possible something could be doing evil stuff it shouldn't.  We can include them, but the output listing returned to nagios is going to be awfully large.  Maybe its best to include them for now and maybe we could trim things down later once we get an idea of what it looks like.